src/current/v24.3/ui-storage-dashboard.md

```diff
@@ -77,24 +83,46 @@ If you are running multiple nodes on a single machine (not recommended), the act
 
 For Windows systems, you can ignore the File Descriptors graph because the concept of file descriptors is not applicable to Windows.
 
+## Disk Write Breakdown
+
+- In the node view, the graph shows the number of bytes written to disk per second categorized according to the source for that node.
+
+- In the cluster view, the graph shows the number of bytes written to disk per second categorized according to the source for each node.
+
+Possible sources of writes with their series label are:
+
+- [WAL]({% link {{ page.version.version }}/architecture/storage-layer.md %}#compaction) (`pebble-wal`)
+- [Compactions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#memtable-and-write-ahead-log) (`pebble-compaction`)
```

Comment:
whereas this one should be to `#compaction`

Reply:
good catch, corrected link